internal/jobqueue/api_test.go

```diff
@@ -37,7 +37,7 @@ func TestBasic(t *testing.T) {
 		registry := distro.NewRegistry([]string{"."})
 		api := jobqueue.New(nil, store.New(nil, distroStruct, *registry))
 
-		test.TestRoute(t, api, false, c.Method, c.Path, c.Body, c.ExpectedStatus, c.ExpectedJSON)
+		test.TestNonJsonRoute(t, api, false, c.Method, c.Path, c.Body, c.ExpectedStatus, &c.ExpectedJSON)
```

Comment:
Why did this and the next call change from `TestRoute` to `TestNonJsonRoute`? What is the difference in behaviour?

For the commit message: how/why are we now ignoring the response in some cases?

Reply:
I've removed this commit and instead properly implemented the tests in the next commit.